### DISCUSSION TITLE
ERROR: "make TARGET=cc2538dk  motelist"

### DISCUSSION THREAD

**Discussion by tlzfighting** [contiki-ng/contiki-ng#2347] [Q&A] [answered]
Hi,sir!  USB devices can be detected using the command "ls /dev/ttyUSB*", but not the command "make TARGET=cc2538dk motelist":
![图片](https://user-images.githubusercontent.com/95647449/208219596-900ed5ed-1dfe-4ddb-99c3-c42272c5d08b.png)

How can I solve this problem? Looking forward to your valuable comments.

**Comment by pjonsson** [comment, marked as answer]
There is help in the Makefile, so `make help` shows you a list of targets. The motelist target is called `motelist-all` rather than motelist.
